build/bin/build-server

```diff
@@ -21,6 +21,7 @@ ${bin_dir}/build-server-rpm percona-qan-app qan-app
 ${bin_dir}/build-server-rpm pmm-server
 ${bin_dir}/build-server-rpm pmm-update
 ${bin_dir}/build-server-rpm dbaas-controller
+${bin_dir}/build-server-rpm dbaas-tools
```

Comment:
And we will merge that line, but not "Submodule pmm-server-packaging updated" below